reconcile/utils/gitlab_api.py

```diff
@@ -662,3 +662,8 @@ def last_comment(
                 continue
             return comment
         return None
+
+    def get_commit_sha(self, ref: str, repo_url: str) -> str:
+        project = self.gl.get_project(repo_url)
+        commits = project.commits.list(ref_name=ref)
```

Comment:
since here we just care about the first one, can limit `list` to only return 1 commit using [per_page](https://python-gitlab.readthedocs.io/en/stable/api-usage.html#pagination).

```python
commits = project.commits.list(ref_name=ref, per_page=1)
```

Reply:
Good idea. Fixed